### ISSUE TITLE
Accessibility: Improperly nested headers

### ISSUE THREAD

**Issue by sarahmhgov** [18F/tock#1567]
Steps to Reproduce:

Location:
https://tock.18f.gov/utilization/analytics

Expected:
The data table collapsed sections should be h4. 

Actual:
The data table collapsed sections are h2 (which is higher in hierarchy than the H3 titles). 

Remediation guidance:
Change the data table section titles to h4. 

<img width="800" alt="image" src="https://user-images.githubusercontent.com/124302726/222297215-0c8a6e1c-10f4-48af-8c04-f282286bc75b.png">

**Comment by jasnakai** [comment]
PR #1583  created for resolution of issue.
